src/common/class-syntax.k

```diff
@@ -30,22 +31,15 @@ syntax ClassMemberDec ::= FieldDec | ClassDec | MethodDec | InterfaceDec | SemiC
 syntax SemiColon ::= ";" [klabel('SemiColon)]
 
 //@ \subsection{ConstructorDeclarations.sdf}
-//todo optionals
-syntax ConstrHead   ::=  AnnoConstrModList TypeParams Type Id "(" Params ")" Throws [prefer, klabel('ConstrDecHead)]
-                       | AnnoConstrModList TypeParams Type Id "(" Params ")"    [klabel('ConstrDecHead)]
-                       | AnnoConstrModList Type Id "(" Params ")" Throws   [klabel('ConstrDecHead)]
-                       | AnnoConstrModList Type Id "(" Params ")"    [klabel('ConstrDecHead)]
+syntax ConstrHead   ::=  AnnoConstrModList OptionalTypeParams Type
+                        Id "(" Params ")" OptionalThrows [klabel('ConstrDecHead)]
```

Comment:
Oh yes, this bit was a question concerned me, there is a one hundred characters limit per line, but sometimes I feel splitting a line into two make it less understandable. What if just the entire production would exceed one hundred characters? Shall I cut it into two lines and where to start the second line so that people won't think I forgot a vertical bar.